sources/packages/backend/libs/services/src/constants/system-configurations-constants.ts

```diff
@@ -0,0 +1,8 @@
+import { Duration } from "zeebe-node";
+
+/**
+ * Default time that a message can wait, once it is delivery to the broker,
+ * to find its correlated key.
+ */
+export const ZEEBE_PUBLISH_MESSAGE_DEFAULT_TIME_TO_LEAVE =
```

Comment:
mm...what is this for. I mean do we need a default time? and what happens if it exceeds the default time